helper/communicator/step_connect.go

```diff
@@ -53,6 +53,7 @@ func (s *StepConnect) Run(state multistep.StateBag) multistep.StepAction {
 			Config:      s.Config,
 			Host:        s.Host,
 			WinRMConfig: s.WinRMConfig,
+			WinRMPort:   s.SSHPort,
```

Comment:
Thats by mistake part of this PR, it actually belongs to #2443. 

It does indeed look wrong, but most of the communicator implementation reuses components of the SSH one, like the SSH step that creates the NAT on virtualbox.